spec/ruby/core/false/to_s_spec.rb

```diff
@@ -4,4 +4,10 @@
   it "returns the string 'false'" do
     false.to_s.should == "false"
   end
+
+  ruby_version_is "2.7" do
+    it "returns a frozen string ''" do
+      false.to_s.frozen?.should == true
+    end
```

Comment:
Could you add a spec that it's always the same String returned, as for Module#name & Symbol#to_s?